pixl_ehr/tests/test_processing.py

```diff
@@ -346,7 +346,6 @@ async def test_radiology_export_multiple_projects(example_messages, tmp_path) ->
         await process_message(mess)
 
     # ACT
```

Comment:
> 👌 second option is to create a separte function `export_reports_to_parquet` and test that instead of testing the REST API call. That feels like less faffing around but I'm easy

I assumed this is what you meant by having `export_radiology_as_parquet` and `send_via_ftps` as separate functions called in a single API endpoint i.e. `export_patient_data` but testing just `export_radiology_as_parquet` in `test_processing` since send via ftps is tested elsewhere

Reply:
System test is failing now because of the call to `export_patient_data()` in `pixl export-patient-data` which in turn calls `send_via_ftps()` (which we want I think!). However, this branch doesn't have an FTPS server set up yet for the system tests (that's under way in #268), so it's failing because of the same reason as before.

I think we can leave this for now and wait until #268 is fixed and merged and then merge `main` into this branch and add an additional check in the system tests to verify the successful upload of the parquet files.